packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -542,6 +542,27 @@ suite('createComponent', () => {
     );
   });
 
+  test('unsets reflected attributes from property with different specifier on null or undefined prop', async () => {
+    // Our types don't allow `ariaChecked` only `aria-checked` but it's possible
+    // for JS users and wrapper will set the property on the element
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    await renderReactComponent({ariaChecked: true} as any);
+
+    // Assertion below fails for Firefox as setting the property doesn't reflect
+    // assert.equal(wrappedEl.hasAttribute('aria-checked'), true);
```

Comment:
`'ariaChecked' in document.body` could be used to skip this test on Firefox. (Returns `true` in Chrome & `false` in safari. This would allow testing instead of using a comment.

Reply:
Excellent suggestion. Thank you!